src/pages/Welcome/style.css

```diff
@@ -1,18 +1,81 @@
+.welcome-container {
+  align-items: center;
+  background-color: #f2b807;
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+  width: 100%;
+}
+
+.welcome-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
 .welcome-form {
-  width: 250px;
+  max-width: 700px;
+  padding: 0 30px;
   text-align: left;
-  margin-top: 50px;
+}
+
+.shoppin-cart-image {
```

Comment:
Same here. 😄

I'm not sure but I believe that line 22 is `shopping-...` and not `shoppin-...`. hahah 😄

Reply:
🙈 OMG, my bad hahaha I already fix the semantic problem  😝